### DISCUSSION TITLE
Welcome to notion2svelte Discussions!

### DISCUSSION THREAD

**Discussion by nvlgzr** [nvlgzr/notion2svelte#1] [Announcements]
# 🦦
## Hey there…
Got **general questions**, interested in **collaboration**, or just want to **say Hi**?

**Comment by ar4s-eth** [comment]
Got started with this last night, thank you for your work. 

I'd say double down on the block customizations and use [svere](https://github.com/FE-PIRL/svere)! There's a bunch of DAO's factories (web3 orgs) integrating Notion-like platforms. 

It would be wicked if there was there was se content unification behind these different initiatives. 

I suppose the focus here is just saving devs the heavy lifting for getting their motion workapaces published tho.

Anyway, thanks again for your work!

**Comment by nvlgzr** [comment]
Hi @ar4s-eth!

Much of what you said is beyond me, lol, but I appreciate the enthusiastic note! Sorry for the delayed response, btw. (I borked my mx records while changing domains, and ended up losing a bunch of notifications from GitHub without realizing it.)

Agree about doubling down on block customizations.

How exactly do you imagine _notion2svelte_ "using svere"? I checked it out, and it seems like a great tool (esp. if you're already stuck in vue/react land), but I can't quite wrap my head around what would make svere a good dependency to have in this project, and what that would look like. That said: it's open source, so if you've got a pull request, I'm game to give it a spin! 😁
